lib/config.js

```diff
@@ -56,7 +60,28 @@ export function configure (otpConfig) {
         </div>
       </div>
     ),
-    LegIcon: TriMetLegIcon,
-    ModeIcon: TriMetModeIcon
+    /**
+     * This function will return a React Element for the main side panel (where
+     * the form elements and itinerary narratives are). This function is run
+     * once on application startup.
+     */
+    getMainPanel: () => {
```

Comment:
I actually kind of like this approach of making the MainPanel a function and putting it in the config. This approach means that the app wouldn't need to import the whole Calltaker or BatchRoutingPanel if it didn't need it which could save on bundle size. The approach of just putting all the logic in the app though is more straightforward and would result in less config file changes. I could go either way on this, but the code should pick one approach or the other.

Reply:
Yea, I initially had coded it up to use config.js, but somehow the assignment of the components to the mainPanel variable in app.js wasn't happening correctly. I agree this is a better way to go, but got tripped up with getting it to work. Does anything stand out to you as being wrong in the `getMainPanel` approach?

Reply:
No not really. The react slots pattern seems to be something people are comfortable with. We can just make this a little clearer when we create better documentation.